htdocs/accountancy/journal/purchasesjournal.php

```diff
@@ -210,9 +210,9 @@
         			$bookkeeping->doc_type = 'supplier_invoice';
         			$bookkeeping->fk_doc = $key;
         			$bookkeeping->fk_docdet = 0;    // Useless, can be several lines that are source of this record to add
-        			$bookkeeping->code_tiers = $tabcompany[$key]['code_fournisseur'];
+        			$bookkeeping->code_tiers = $tabcompany[$key]['code_compta_fournisseur'];
```

Comment:
the code_tiers is the customer code here. The accounting account is inside numero_compte